content/en/docs/concepts/configuration/configmap.md

```diff
@@ -221,6 +221,54 @@ in a Pod:
    in the specified environment variables.
 
 This is an example of defining a ConfigMap as a pod environment variable:
+
+The following ConfigMap (myconfigmap.yaml) stores two properties: username and access_level:
```

Comment:
Even better (just so you know): use a `code_sample` shortcode.

Or, if not doing that:
```diff
-The following ConfigMap (myconfigmap.yaml) stores two properties: username and access_level:
+The following ConfigMap (which you might name `myconfigmap.yaml`) stores two properties:
+`username` and `access_level`:
```